src/sentry/api/serializers/models/incident.py

```diff
@@ -51,6 +87,13 @@ def serialize(self, obj, attrs, user):
 
 
 class DetailedIncidentSerializer(IncidentSerializer):
+    def __init__(self, expand=None):
+        if expand is None:
+            expand = ["seen_by"]
+        elif "seen_by" not in expand:
+            expand.append("seen_by")
+        self.expand = expand
```

Comment:
`super().__init__(expand=expand)`

Reply:
Could you change this line? I'd prefer to just let the superclass set this rather than setting it ourselves.